source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```diff
@@ -0,0 +1,2392 @@
+    );
+    background-position: 0 0;
+    background-size: 100% 100%;
+    border-radius: 15px;
+    
+    /* Interactive */
+    cursor: pointer;
+  }
+  
+  .caButton a {
+    text-decoration: none;
+    color: var(--support-popup-text);
+    cursor: pointer;
+  }
+  
+  .caButton:hover {
+    background: var(--unraid-66-color);
+    color: #fff !important;
+  }
+  
+  .caButton:hover a,
+  a.caButton:hover {
+    color: #fff;
+    text-decoration: none;
+  }
+  
+  .maxPerPage {
+    position: absolute;
+    right: 0;
+  }
+  
+  #dropdown-maxPerPage {
+    position: absolute;
+    right: 0;
+  }
+
+  
+  /* Form Elements */
+  input[type="checkbox"] {
+    width: 2rem;
+    height: 2rem;
+    margin-left: 0;
+  }
+  
+  input[type="button"] {
+    background: none;
+    font-size: 1.5rem;
+  }
+  
+  input:hover[type="button"] {
+    color: var(--unraid-66-color) !important;
+    background: none !important;
+  }
+  
+  input:hover[type="button"][disabled] {
+    background: none !important;
+    color: currentColor !important;
+  }
+  
+  input[type="button"][disabled] {
+    background: none !important;
+  }
+  
+  .searchSubmit {
+    /* Layout */
+    width: 2rem;
+    min-width: 2rem;
+    height: 3.4rem;
+    padding: 1.1rem 1rem 1rem 0;
+    position: relative;
+    
+    /* Typography */
+    font-family: 'FontAwesome';
+    font-size: 1.1rem;
+    
+    /* Visual */
+    background: var(--template-hover-background);
+    border: none;
+    
+    /* Interactive */
+    cursor: pointer;
+  }
+  
+  #searchButton:hover {
+    color: var(--unraid-66-color);
+  }
+  
+  /* Menu and Navigation */
+  .menuHeader {
+    font-size: 2rem;
+    margin: 1rem 0;
+  }
+  
+  .menuItems {
+    position: relative;
+    top: 2rem;
+    height: auto;
+    align-items: stretch;
+    min-width: 14rem;
+    max-width: 14rem;
+  }
+  
+  .selectedMenu {
+    color: var(--unraid-66-color);
+    font-weight: bold;
+  }
+  
+  .hoverMenu {
+    color: var(--unraid-66-color) !important;
+  }
+  
+  /* Sidebar Navigation */
+  .sidebar {
+    /* Layout */
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 1.6rem;
+    width: 100%;
+    height: 100vh;
+    margin-bottom: 10px;
+    /* Display over any banner warning */
+    z-index: 1000;
+    opacity: 0.9;
+    
+    /* Visual */
+    background-color: var(--sidebar-close-background);
+    display: none;
+  }
+  
+  .sidebarClose {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+  }
+  
+  .sidenav {
+    /* Layout */
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 1.6rem;
+    margin-bottom: 10px;
+    padding-top: 60px;
+    
+    /* Visual */
+    background-color: var(--sidebar-background);
+    color: var(--sidebar-text);
+    
+    /* Interactive */
+    overflow-x: hidden;
+    overflow-y: scroll;
+    transition: 0.5s;
+  }
+  
+  .sidenavHide {
+    width: 0;
+  }
+  
+  .sidenavShow {
+    width: 90%;
+    max-width: 70rem;
+  }
```

Comment:
CSS transitions for `width` a very taxing and prone to frame rate lag.

Consider keeping the width consistent and instead use `transform: translate()` and opacity to determine visibility of this element.